nodes/voice_control_updated.py

```diff
@@ -62,13 +66,22 @@ def start_recognizer(self):
         #Keyword list file for keyword searching
         config.set_string('-kws', self.kw_list)
 
-        stream = pyaudio.PyAudio().open(format=pyaudio.paInt16, channels=1,
-                        rate=16000, input=True, frames_per_buffer=1024)
+        rospy.loginfo("Opening the audio channel")
+
+	# I recommend installing and running audacity to help figure this out
+	# Other useful commands:
+	# pactl list short sources
+	# pacmd list-sinks
+        stream = pyaudio.PyAudio().open(format=pyaudio.paInt16, channels=2,
+                        rate=44100, input=True, frames_per_buffer=65536)
```

Comment:
What effect did changing the values to these specific constants have?
Channels I can understand, but others?